packages/flutter/test/services/raw_keyboard_test.dart

```diff
@@ -538,13 +538,13 @@ void main() {
         'toolkit': 'glfw',
         'keyCode': 65,
         'scanCode': 0x00000026,
-        'codePoint': 97,
+        'unicodeScalarValuesProduced': 113,
```

Comment:
Can you also add back in some tests that test two and three scalar values encoded in the `unicodeScalarValuesProduced` member?  (the "three" scenario should assert).

Reply:
Done